tests/unit/views/CartView.spec.ts

```diff
@@ -193,26 +114,33 @@ describe('CartView.vue', () => {
   })
 
   it('adds open menu items', async () => {
-    // @ts-ignore
-    const wrapper = mount(CartView, { stubs, store, localVue, mocks })
-
+    const wrapper:any = mount(CartView, { stubs, store, localVue, mocks })
     const cartHeader = wrapper.find('.hoverable')
     cartHeader.trigger('click')
-
     // the first element closes on click.
     expect(wrapper.vm.openItems).toHaveLength(0)
-
     cartHeader.trigger('click')
     // the element is openened
     expect(wrapper.vm.openItems).toHaveLength(1)
   })
 
   it('can render variables sets correctly', async () => {
-    // @ts-ignore
-    const wrapper = mount(CartView, { stubs, store, localVue, mocks })
-    // @ts-ignore
+    const wrapper:any = mount(CartView, { stubs, store, localVue, mocks })
     expect(wrapper.vm.variableSetClass(cartTree[0].subsections[0].variables[0])).toEqual('start')
     expect(wrapper.vm.variableSetClass(cartTree[0].subsections[0].variables[1])).toEqual('line')
     expect(wrapper.vm.variableSetClass(cartTree[0].subsections[0].variables[2])).toEqual('end')
   })
+
+  it('select all available variables', async () => {
+    const wrapper = shallowMount(CartView, { stubs, localVue, store, mocks })
+    const toggleAllButton = wrapper.find('.t-toggle-all-variables')
+    toggleAllButton.trigger('click')
+    await wrapper.vm.$nextTick()
+    await wrapper.vm.$nextTick()
```

Comment:
why are there 2 ? , add comment or remove

Reply:
It takes two ticks to update the Vuex state. Added a comment.